packages/core/android/src/main/kotlin/src/DdSdkReactNativePackage.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative
+
+import com.facebook.react.TurboReactPackage
+import com.facebook.react.bridge.NativeModule
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.module.model.ReactModuleInfo
+import com.facebook.react.module.model.ReactModuleInfoProvider
+
+/**
+ * Package of native dd-sdk-reactnative native modules.
+ */
+class DdSdkReactNativePackage : TurboReactPackage() {
+    override fun getModule(name: String, reactContext: ReactApplicationContext): NativeModule? {
+        return if (name == DdSdkImplementation.NAME) {
+            DdSdk(reactContext)
+        } else if (name == DdRumImplementation.NAME) {
+            DdRum(reactContext)
+        } else if (name == DdTraceImplementation.NAME) {
+            DdTrace(reactContext)
+        } else if (name == DdLogsImplementation.NAME) {
+            DdLogs(reactContext)
+        } else {
+            null
+        }
+    }
+
+    override fun getReactModuleInfoProvider(): ReactModuleInfoProvider {
+        return ReactModuleInfoProvider {
+            val moduleInfos: MutableMap<String, ReactModuleInfo> =
+                HashMap()
```

Comment:
probably can be just the following:

```suggestion
            val moduleInfos = mutableMapOf<String, ReactModuleInfo>()
```